### DISCUSSION TITLE
stories_things/basic_content/static/

### DISCUSSION THREAD

**Discussion by giscus** [Light-City/light-comments#5] [Announcements]
# stories_things/basic_content/static/

Stories About C Plus Plus

https://light-city.github.io/stories_things/basic_content/static/

**Comment by BAI-HANK** [comment]
perfect , thank you  pay for time to  writed it

**Comment by deeprado** [comment]
第二个案子，需要对静态变量初始化吧，我这报错了 。 int Apple::i = 1;
